packages/scratch-vm/src/extensions/scratch3_musiccreation/index.js

```diff
@@ -561,8 +561,7 @@ class Scratch3MusicCreation {
      * @property {int} INSTRUMENT - the number of the instrument to select.
      */
     setInstrumentForBelow(args, util) {
-        const instrument = this.musicCreationHelper.getInstrumentValue(args.INSTRUMENT);
-        addTopBlockModifier(util, args[internalIDKey], instrumentModifierKey, instrument);
+        addTopBlockModifier(util, args[internalIDKey], instrumentModifierKey, Cast.toNumber(args.INSTRUMENT));
```

Comment:
The the original `getInstrument` was doing more rounding (probably just for safety) than just casting:

```js
getInstrumentValue(instNum) {
        instNum = Cast.toNumber(instNum);
        instNum = Math.round(instNum);
        const stage = this.runtime.getTargetForStage();
        if (stage) {
            stage.instrument = this.findInstrumentForNumber(instNum);
        }
        instNum -= 1; // instruments are one-indexed
        return MathUtil.wrapClamp(instNum, 0, this.INSTRUMENT_INFO.length - 1);
    }
```

Do you think it'd be worth it to implement a new function in `musiccreationhelpers` that captures all that functionality? or not worth it?